<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PearRay CNT Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../utils/resources/icons.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="viewWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="View3DWidget" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenScene"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuExternal_Links">
     <property name="title">
      <string>External Links</string>
     </property>
     <addaction name="actionWebsite"/>
    </widget>
    <addaction name="actionShortcuts"/>
    <addaction name="actionQuickGuide"/>
    <addaction name="actionManual"/>
    <addaction name="separator"/>
    <addaction name="menuExternal_Links"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nodes:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="nodeCountLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Inner Nodes:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="innerNodeCountLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Max Depth:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="maxDepthLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>BoundingBox:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::DashLine</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <row>
           <property name="text">
            <string>Low</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Up</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>X</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Y</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Z</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>252</red>
              <green>175</green>
              <blue>62</blue>
             </color>
            </brush>
           </property>
          </item>
          <item row="0" column="1">
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>114</red>
              <green>159</green>
              <blue>207</blue>
             </color>
            </brush>
           </property>
          </item>
          <item row="0" column="2">
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>173</red>
              <green>127</green>
              <blue>168</blue>
             </color>
            </brush>
           </property>
          </item>
          <item row="1" column="0">
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>252</red>
              <green>175</green>
              <blue>62</blue>
             </color>
            </brush>
           </property>
          </item>
          <item row="1" column="1">
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>114</red>
              <green>159</green>
              <blue>207</blue>
             </color>
            </brush>
           </property>
          </item>
          <item row="1" column="2">
           <property name="background">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>173</red>
              <green>127</green>
              <blue>168</blue>
             </color>
            </brush>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Max Depth:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="depthLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="buddy">
        <cstring>maxDepthSlider</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="maxDepthSlider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionQuickGuide">
   <property name="text">
    <string>Quick Guide</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionWebsite">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_home_black_24dp_2x.png</normaloff>:/ic_home_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Website</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionOpenScene">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/ic_nature_people_black_24dp_2x.png</normaloff>:/ic_nature_people_black_24dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>Open scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>View3DWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>widgets/View3DWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../utils/resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>maxDepthSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>depthLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
